extension/js/tree.js

```diff
@@ -2,6 +2,11 @@
  *	DeepCite Source Tree page 
 */
 
+const url = "https://us-central1-deepcite-306405.cloudfunctions.net/deepcite";
```

Comment:
Eventually, do you think these could be environment variables that are loaded in when building the code?

Reply:
https://stackoverflow.com/questions/196498/how-do-i-load-the-contents-of-a-text-file-into-a-javascript-variable

Reply:
A good idea. I'll look into doing this